src/train_model.py

```diff
@@ -19,10 +19,6 @@
 from torchinfo import summary as model_summary
 from utils import Struct
 
-# Allow torch to run float32 matrix multiplications in lower precision for
-# better performance while training if hardware is capable
-torch.backends.cuda.matmul.allow_tf32 = True
```

Comment:
I assume that precision in trainer covers this instead, right?

Reply:
yup